app/views/layouts/blacklight/base.html.erb

```diff
@@ -53,5 +53,37 @@
 
     <%= render partial: 'shared/footer' %>
     <%= render partial: 'shared/modal' %>
+
+<script>
+  $(function() {
+    // Returns the value for a given query string parameter.
+    // Source: https://css-tricks.com/snippets/javascript/get-url-variables/
+    var queryStringParam = function(name) {
+      var query = window.location.search.substring(1);
+      var vars = query.split("&");
+      for (var i=0; i<vars.length; i++) {
+        var pair = vars[i].split("=");
+        if (pair[0] == name){
+          return pair[1];
+        }
+      }
+      return(false);
+    }
+
+    // Returns the active facet value for a given field
+    var activeFacet = function(fieldName) {
+      return queryStringParam(encodeURI("f[" + fieldName + "][]"));
+    }
+
+    var hideFacetGroup = function(fieldName) {
+      $(`.blacklight-${fieldName}`).remove();
+    }
+
+    // Hide the subcommunity facet unless we are faceting by community or subcommunity.
+    if (!activeFacet("community_root_name_ssi") && !activeFacet("subcommunity_name_ssi")) {
+      hideFacetGroup("subcommunity_name_ssi");
+    }
```

Comment:
This is the code that hides the subcommunity facet unless it's relevant.

Reply:
I love that you left a comment about where the inspiration for this came from. Very useful for long-term maintenance and kind to your colleagues!